spongeauth/spongeauth/settings/prod.py

```diff
@@ -63,10 +59,11 @@
     d = SSO_ENDPOINTS.setdefault(name, {})
     d[key.lower()] = v
 
-RAVEN_CONFIG = {
-    'dsn': os.environ['RAVEN_DSN'],
-    'release': raven.fetch_git_sha(GIT_REPO_ROOT),
-}
+sentry_sdk.init(
+    dsn=os.environ['RAVEN_DSN'],
```

Comment:
I could rename this one to `RAVEN_DSN`. Would need a change in the env settings of the production machine.

Reply:
Slight preference for 

```suggestion
    dsn=os.environ.get('RAVEN_DSN'),
)
```

Specifically, it's useful to have usernames/user IDs when debugging issues with auth, and also if you use `.get` then we can easily swap RAVEN_DSN to SENTRY_DSN.